### PR TITLE
ListItem secondaryText color

### PR DESCRIPTION
**Pull request by mgonzalezs**
Fix for https://github.com/codypearce/material-bread/issues/199
- Added secondaryTextStyle prop to ListItem component
- Removed unnecessary styles from `ListItem.styles.js `

### CODE REVIEW

**Comment by codypearce** [comment]
@mgonzalezs Thanks! This prop should be added to the docs, let me know if you'd like me to do that instead.

**Comment by mgonzalezs** [comment]
Done @codypearce . Let me know if the changes are ok. Thanks

File: docs/src/content/components/list-item/propData.js [hunk 1]
```diff
@@ -14,10 +14,11 @@ const propData = [
     'node',
     '',
   ],
-
+  ['listItemText', 'Styles main text element', 'object', ''],
```
Comment (codypearce) [inline docs/src/content/components/list-item/propData.js]: I think you mean `textStyle`

Comment (mgonzalezs) [reply docs/src/content/components/list-item/propData.js]: Totally, my bad. Fixed

**Comment by codypearce** [comment]
@mgonzalezs Thanks!

**Pull request merged**